app/views/layouts/_navbar.html.erb

```diff
@@ -1,55 +1,93 @@
-<nav class="navbar navbar-inverse" data-turbolinks-permanent>
-  <div class="container-fluid">
-    <div class="navbar-header">
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-2">
-        <span class="sr-only">Toggle navigation</span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-      </button>
-      <a class="navbar-brand lobster" href="/">CoPosition</a>
-    </div>
-
-    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-2">
-      <ul class="nav navbar-nav">
-        <li><a href="<%= signed_in? ? developers_console_path : new_developer_session_path %>">Developers</a></li>
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">About <span class="caret"></span></a>
-          <ul class="dropdown-menu" role="menu">
-            <li><a href="#">Security</a></li>
-            <li><a href="#">Enrolled companies</a></li>
-            <li><a href="#">Why use CoPosition</a></li>
-            <li class="divider"></li>
-            <li><%= link_to("API", api_path) %></li>
-          </ul>
-        </li>
+<nav class="nav-wrapper red lighten-1" role="navigation" data-turbolinks-permanent>
```

Comment:
@GarySiu @limzykenneth 
Hi guys,
Not sure if I'm missing the point, but just going to explain what I'm doing (as it's basically the same thing)

I'm using Materialize, and the normal asset pipeline (using the the materialize-sass gem)
In `application.scss` I have

```css
@import "materialize/components/color";
$primary-color: color("deep-orange", "lighten-1") !default;
@import 'materialize';
```
I haven't needed to make any changes to other Materialize classes.

Again, not sure if this is completely unhelpful.